File: library/src/main/java/com/github/javiersantos/appupdater/objects/GitHub.java
package com.github.javiersantos.appupdater.objects;

public class GitHub {
    private String gitHubUser;
    private String gitHubRepo;

    public GitHub(String gitHubUser, String gitHubRepo) {
        this.gitHubUser = gitHubUser;
        this.gitHubRepo = gitHubRepo;
    }

    public String getGitHubUser() {
        return gitHubUser;
    }

    public void setGitHubUser(String user) {
        this.gitHubUser = user;
    }

    public String getGitHubRepo() {
        return gitHubRepo;
    }

    public void setGitHubRepo(String repo) {
        this.gitHubRepo = repo;
    }

    public static Boolean isGitHubValid(GitHub gitHub) {
        if (gitHub == null || gitHub.getGitHubUser().length() == 0 || gitHub.getGitHubRepo().length() == 0) {
            return false;
        } else {
            return true;
        }
    }

}

File: library/src/main/java/com/github/javiersantos/appupdater/objects/GitHub.kt
package com.github.javiersantos.appupdater.objects

class GitHub(var gitHubUser: String,
             var gitHubRepo: String) {

    companion object {
        fun isGitHubValid(gitHub: GitHub?): Boolean {
            return gitHub != null && gitHub.gitHubUser.isNotEmpty() && gitHub.gitHubRepo.isNotEmpty()
        }
    }
}